notebooks/nvidia_haystack_medical_chatbot.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    !pip install nvidia-haystack==0.0.1
</pre><p>I believe you dont need the exclamation mark here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>